cmd/build/v2/build.go

```diff
@@ -167,16 +185,37 @@ func (bc *OktetoBuilder) Build(ctx context.Context, options *types.BuildOptions)
 
 			buildSvcInfo := buildManifest[svcToBuild]
 
+			// create the meta pointer and append it to the analytics slice
+			meta := analytics.NewImageBuildMetadata()
+			buildsAnalytics = append(buildsAnalytics, meta)
+
+			meta.Name = svcToBuild
+			meta.RepoURL = bc.Config.GetAnonymizedRepo()
```

Comment:
in the issue description it says `should be hashed and not readable` . Do we need to hash the value for `name` and `repoUrl`(already anonymized but not hashed) ?

Reply:
i checked that when running the debugger i get the file root of the repository, but when running binary is the repo.

Reply:
the hashing is done in the analyrics https://github.com/okteto/okteto/pull/4001/files#diff-5d815810b1f75652c01329c28da0b4fa4c0720e47d31f2c5447c19c52333deb6R37